server/src/main/java/org/apache/iotdb/db/conf/IoTDBConstant.java

```diff
@@ -102,6 +102,13 @@ private IoTDBConstant() {
   public static final String PATH_WILDCARD = "*";
   public static final String TIME = "time";
 
+  //sdt parameters
+  public static final String LOSS = "loss";
```

Comment:
The problem is... with Antlr, all these keywords will be converted to lowercase, but if users call thrift API, createTimeseries(), how they know they must write these keywords as lowercase.
(The example in your SQL grammar doc is uppercase)

Reply:
and make sure it works on the cluster mode.

Reply:
Hi, in CreateTimeSeriesPlan, I converted props to case insensitive. 
this.props = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);

Reply:
OK..